### DISCUSSION TITLE
hello could this be used for a console on ace editor

### DISCUSSION THREAD

**Discussion by ldijkman** [azu/codemirror-console#22] [Q&A]
hello could this be used for a console on ace editor

have been searching for demos 
but cannot find a full codemirror with html previewer and console
but i do not fully understand it

i want to add a console float div
that caches the console messages
but stil leaves it printing in dev tools

i think your code does that

https://ldijkman.github.io/Ace_Seventh_Heaven/Hell.html

![2023-10-23-181810_1920x1080_scrot](https://github.com/azu/codemirror-console/assets/45427770/bc69881d-0acd-4e79-97be-8dad50ec60da)

**Comment by azu** [comment]
No, It is for [CodeMirror 5](https://codemirror.net/5/).
